src/org/mozilla/javascript/NativeError.java

```diff
@@ -379,42 +379,12 @@ protected int findPrototypeId(String s)
         private int stackTraceLimit = DEFAULT_STACK_LIMIT;
         private Function prepareStackTrace;
 
-        public Object getStackTraceLimit(Scriptable thisObj) {
```

Comment:
Why remove this function (and the next three)? I'm pretty sure that they're part of the stack trace customization features, if I recall.

Reply:
I had seen that the methods are not used, I'll add at next commit